vllm/distributed/device_communicators/shm_broadcast.py

```diff
@@ -21,6 +23,12 @@
 
 logger = init_logger(__name__)
 
+# We prefer to use os.sched_yield as it results in tighter polling loops,
+# measured to be around 3e-7 seconds. However on earlier versions of Python
+# os.sched_yield() does not release the GIL, so we fall back to time.sleep(0)
+USE_SCHED_YIELD = ((sys.version_info[:3] >= (3, 11, 1))
+                   or (sys.version_info[:3] >= (3, 10, 8)))
```

Comment:
In theory this would match 3.11.0 otherwise I think...
```suggestion
USE_SCHED_YIELD = ((sys.version_info[:3] >= (3, 11, 1))
                   or (sys.version_info[:2] == (3, 10) and sys.version_info[2] >= 8))
```

Reply:
thanks for the catch!